t/porting/authors.t

```diff
@@ -29,12 +29,13 @@ elsif( $ENV{GITHUB_ACTIONS} && length $ENV{GITHUB_BASE_REF} ) {
     # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables
 
     # we should be on a merge commit, but double check
-    my $branch_head = `git rev-parse "HEAD^2" 2>/dev/null`;
+    my $null = $^O eq 'MSWin32' ? 'nul' : '/dev/null/';
```

Comment:
Spurious trailing slash on `/dev/null/`. But it's better to use `File::Spec->devnull` for max portability.

Reply:
Fixed, thanks